scss/angular/overrides/_uib-datepicker.scss

```diff
@@ -0,0 +1,137 @@
+[uib-datepicker-popup] {
+  border-color: $input-border;
+
+  .form-control {
+    border-color: $input-border;
+  }
+}
+
+[uib-datepicker-popup-wrap] {
+  + .input-group-btn > button {
+    border-color: $input-border;
+    color: $crm-color-neutral;
+  }
+}
+
+.uib-left,
+.uib-right,
+.uib-title {
```

Comment:
now that i think about it, these classes: `.uib-left`, `.uib-right`, .uib-title` seem extremely generic, not even related to the datepicker. Are you sure these classes are not used in any of the other UI Bootstrap modules? Let's not find out only later that we are breaking the style of other uib stuff

Reply:
I just checked, its not used in any other directives. But yeah the naming is confusing.